accounts/abi/abi.go

```diff
@@ -166,6 +166,9 @@ func (abi ABI) Call(executer Executer, v interface{}, name string, args ...inter
 }
 
 var interSlice = reflect.TypeOf([]interface{}{})
+var r_hash = reflect.TypeOf(common.Hash{})
+var r_bytes = reflect.TypeOf([]byte{})
+var r_byte = reflect.TypeOf(byte(0))
```

Comment:
Best I can come up with is `reflectHash`, `reflectBytes`. Not much better tbh.

Also if you group the vars into var ( ... ), it will align them nicely.